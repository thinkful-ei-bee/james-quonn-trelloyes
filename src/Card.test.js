import React from "react";
import ReactDOM from "react-dom";
import Card from "./Card";
import renderer from "react-test-renderer";

it("renders without crashing", () => {
  // Create a DOM element to render the component into
  const div = document.createElement("div");

  // Render the component
  // If something is wrong it will fail her
  ReactDOM.render(<Card ></Card>, div);
  // Clean up
  ReactDOM.unmountComponentAtNode(div);
});

// describe("<Card />", () => {
//   it("renders div with a p, button, p and h3 nested inside", () => {
//     // Render the component, as JSON
//     const tree = renderer.create(<Card ><p></p></Card>).toJSON();
//     // Check whether it matches the previous snapshot
//     // Stored in __snapshots__/App.test.js.snap
//     expect(tree).toMatchSnapshot();
//   });
// });
