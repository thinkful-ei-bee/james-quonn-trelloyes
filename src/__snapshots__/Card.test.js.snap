// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> renders div with a p, button, p and h3 nested inside 1`] = `
<div
  className="Card"
>
  <p />
  <button
    onClick={[Function]}
    type="button"
  >
    delete
  </button>
  <h3 />
  <p />
</div>
`;

exports[`<Card /> renders without crashing 1`] = `
<div
  className="Card"
>
  <p />
  <button
    onClick={[Function]}
    type="button"
  >
    delete
  </button>
  <h3 />
  <p />
</div>
`;
